yarn.lock

```diff
@@ -11176,4 +11176,4 @@ yauzl@2.4.1:
   resolved "https://registry.yarnpkg.com/yauzl/-/yauzl-2.4.1.tgz#9528f442dab1b2284e58b4379bb194e22e0c4005"
   integrity sha1-lSj0QtqxsihOWLQ3m7GU4i4MQAU=
   dependencies:
-    fd-slicer "~1.0.1"
+    fd-slicer "~1.0.1"
```

Comment:
Why remove the last line?

Reply:
Not removed intentionally, after installing packages yarn.lock got changed and added in check in.

Reply:
Could you revert the change since you do not actually change anything?

Reply:
Done